src/main/java/uk/gov/companieshouse/insolvency/data/api/InsolvencyApiService.java

```diff
@@ -80,6 +87,15 @@ private ChangedResource mapChangedResource(String contextId,
         changedResource.setResourceKind("company-insolvency");
         changedResource.setContextId(contextId);
 
+        if (isDelete) {
+            try {
+                changedResource.setDeletedData(objectMapper.writeValueAsString(insolvencyDocument));
+            } catch (JsonProcessingException exp) {
```

Comment:
Are you sure we need to convert it to string, deletedData field is an Object type, are we doing the conversion automatically later on?

Reply:
Its a String @amobarak